ouroboros-consensus-shelley-test/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -109,8 +113,9 @@ prop_simple_real_tpraos_convergence TestSetup
     initialKESPeriod :: SL.KESPeriod
     initialKESPeriod = SL.KESPeriod 0
 
-    maxKESEvolution :: Word64
-    maxKESEvolution = 100 -- TODO
+    maxKESEvolutions :: Word64
```

Comment:
Comment: a KES algorithm supports a particular max number of KES evolutions, but we can configure a potentially lower max for the ledger. Here, we just use the same value for both.